test/e2e/density.go

```diff
@@ -210,6 +214,7 @@ var _ = Describe("Density", func() {
 				PodStatusFile:        fileHndl,
 				Replicas:             totalPods,
 				MaxContainerFailures: &MaxContainerFailures,
+				Timeout:              time.Duration(math.Ceil(float64(itArg.podsPerNode)/minPodStartupThroughput)) * time.Minute,
```

Comment:
BTW, this is 15 pods per *minute*. The apiserver qps limit in kubelet is still capped at 5 per second, so this is going to be slower for kubelet to report as the number of pods scales.